src/components/StalkCalculator.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState, useEffect} from 'react';
+
+const StalkCalculator = () => {
+
+    //set state variables and values
```

Comment:
I wish I had something useful to add here, I'm just really impressed!! Great use of useState instead of relying on class-based components. Also, a very topical calculator choice :)

Reply:
Thanks, LauraJBrennan!  I love Animal Crossing!~  I have just recently been messing with hooks and this seemed like a great opportunity to apply it!